config.js

```diff
@@ -252,6 +252,14 @@ var config = {
     // Default value for the channel "last N" attribute. -1 for unlimited.
     channelLastN: -1,
 
+    // Connection indicators
+    // connectionIndicators: {
```

Comment:
Please remove the legacy options from interface_config.js . Just leave a comment saying they got moved to config.js.